search-app/docs/api.md

```diff
@@ -1589,6 +1590,18 @@ Find collections that match all of the 'consortium' param values
 
      curl "%CMR-ENDPOINT%/collections?consortium\[\]=CWIC&consortium\[\]=FEDEO&consortium\[\]=CEOS&options\[consortium\]\[and\]=true"
 
+#### <a name="c-latency"></a> Find collections by latency
```

Comment:
There is no way anyone using this API will know what the 1 of 3 `latency` values to use to search for collections. There should be introductions on the search parameter. 
Also I think we should just use the term `collection_data_type` in Collection metadata as the parameter name and the values that are defined in the schema for the search. `latency` is something we can use in facet result to help user understand it, but not in searching.

Reply:
I think we need to add a new one. Otherwise the facet will show not only "1 to 3 hours" but also "NEAR_REAL_TIME".   We only want to show "1 to 3 hours."

Reply:
The latency search parameter does not make sense to the clients. We can keep it internal for v2 facets and not show it in API doc similar to what we did with another v2 facets field `horizontal-data-resolution-range`. We can remove this section as `collection_data_type` parameter is already documented in API doc.

Reply:
Sure. That makes sense.  Removed.

Reply:
We've decided to keep the new latency index.